README.md

```diff
@@ -59,6 +61,12 @@ See [react-ui](https://github.com/CrossMint/crossmint-client-sdk/tree/main/packa
 
 ---
 
+## Migration guide to 0.1.0 version
+
+If you are already using Crossmint and you are not sure how to upgrade to the 0.1.0 version, [checkout the migration guide](https://docs.google.com/document/d/14IKpjrij7kU7Dr0I7rZkf0PyDNbXiklx2v4GuzUrFbw/edit?usp=sharing).
```

Comment:
Version 0.1.0 introduces breaking changes. To learn how to migrate from a version lower than 0.1.0, check out the migration guide.